apps/admin/src/components/customLayouts/CollapsibleFormSegment.tsx

```diff
@@ -0,0 +1,46 @@
+import { CollapsibleCard, H5, ParMd } from '@daohaus/ui';
+
+type SegmentType = {
+  actionButton?: React.ReactNode;
+  collapsible?: boolean;
+  defaultOpen?: boolean;
+  title?: string | React.ReactNode;
+  description?: string | React.ReactNode;
+  formArea: React.ReactNode;
+  infoArea?: React.ReactNode;
+  error?: React.ReactNode;
+};
+
+export const CollapsibleFormSegment = ({
+  actionButton,
+  defaultOpen,
+  title,
+  description,
+  formArea,
+  infoArea,
+  error,
+}: SegmentType) => {
+  return (
+    <CollapsibleCard
```

Comment:
Does this still have the card BG and stuff like before? I think it would worthwhile to make it look the same as the other forms if not.  Could be a separate ticket.

Reply:
yes, it holds the same properties